modules/ROOT/pages/traversal-framework/traversal_framework_java_api.adoc

```diff
@@ -0,0 +1,353 @@
+:description: The Neo4j Traversal Framework Java API.
+
+:org-neo4j-graphdb-Direction-both: {neo4j-javadocs-base-uri}/org/neo4j/graphdb/Direction.html#BOTH
+:org-neo4j-graphdb-traversal-description: {neo4j-javadocs-base-uri}/org/neo4j/graphdb/Transaction.html#traversalDescription()
+
+[[traversal-java-api]]
+= Traversal Framework Java API
+
+[[traversal-java-api-traversaldescription]]
+== link:{org-neo4j-graphdb-traversal-description}[TraversalDescription^]
```

Comment:
This link is broken when I built the docs locally, `TraversalDescription` is not in the javadocs. Do you know if it will appear once the code has been undeprecated from the database?